File: aeongseo/SWEA/CPP/swea2071.cpp
/*** 2071. 평균값 구하기 ***/


#include<iostream>
using namespace std;

int main() {
	int test_case;
	int T;

	cin >> T;

	for (test_case = 1; test_case <= T; ++test_case) {
		int arr[10];
		int sum = 0, avg = 0;
		for (int i = 0; i < 10; i++) {
			cin >> arr[i];
		}

		for (int j = 0; j < 10; j++) {
			sum += arr[j];
		}

		if (sum % 10 >= 5) {                            // 평균의 소수점이 0.5 이상일 때 올림
			avg = sum / 10 + 1;
		}
		else {                                          // 0.5 미만이면 버림
			avg = sum / 10;
		}

		cout << '#' << test_case << ' ' << avg << '\n';
	}

	return 0;
}

File: aeongseo/SWEA/PYTHON/swea2071.py
'''
2071. 평균값 구하기
'''

T = int(input())

for test_case in range(1, T+1):
    arr = list(map(int, input().split()))
    sum = 0

    for i in range(10):
        sum += arr[i]               # 모든 값의 합
    
    if sum % 10 >= 5:               # 평균의 소수점이 0.5 이상일 때 올림
        avg = int(sum / 10) + 1     
    else:
        avg = int(sum / 10)         # 0.5 미만이면 버림
    
    print(f'#{test_case} {avg}')